src/resources/index.js

```diff
@@ -173,6 +173,7 @@ module.exports = {
         'the version of used OC CLI is invalid. Try to upgrade OC CLI running {0}',
       PACKAGE_CREATION_FAIL: 'An error happened when creating the package: {0}',
       PACKAGING_FAIL: 'an error happened while packaging {0}: {1}',
+      PACKAGE_FOLDER_MISSING: 'Could not find a _package folder to publish',
```

Comment:
Would it be worth to start adding hints? ex. "make sure to run `oc package` first or drop the `--skip-package` parameter"

Reply:
sure. done